content/docs/tutorials/getting-started-with-cert-manager-on-azure-kubernetes-service-using-lets-encrypt-for-ingress-ssl/README.md

```diff
@@ -0,0 +1,397 @@
+description: Learn how to deploy cert-manager on Azure Kubernetes Service (AKS) and then configure it to sign SSL certificates using Let's Encrypt
+---
+
+*Last Verified: 25 November 2022*
+
+In this tutorial you will learn how to deploy and configure cert-manager on Azure Kubernetes Service (AKS).
+You will learn how to configure cert-manager to get a signed SSL certificate from Let's Encrypt,
+using an [DNS-01 challenge](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge).
+Finally you will learn how the certificate can be used to serve an HTTPS website with a public domain name.
+
+> **Microsoft Azure**: A suite of cloud computing services by Microsoft.<br/>
+> **Kubernetes**: Runs on your servers. Automates the deployment, scaling, and management of containerized applications.<br/>
+> **cert-manager**: Runs in Kubernetes. Obtains TLS / SSL certificates and ensures the certificates are valid and up-to-date.<br/>
+> **Let’s Encrypt**: An Internet service. Allows you to generate free short-lived SSL certificates.
+
+## 0. Configure the Azure CLI (az)
+
+If your have not already done so, you may want to set up the `az` command for interactive use:
+
+> ℹ️ Choose "Optimize for interaction" when prompted.
+
+```bash
+az init
+```
+
+Install the [Azure CLI AKS Preview Extension], which you will need to configure some advanced workload identity federation features on your AKS cluster.
+
+```bash
+az extension add --name aks-preview
+```
+
+[Azure CLI AKS Preview Extension]: https://github.com/Azure/azure-cli-extensions/tree/main/src/aks-preview
+
+
+Log in, if you have not already done so:
+
+```bash
+az login
+```
+
+> ℹ️ A web browser will be opened at https://login.microsoftonline.com/organizations/oauth2/v2.0/authorize.
+> Continue the login in the web browser and then return to your terminal.
+
+Set the default resource group and location:
+
+```bash
+export AZURE_DEFAULTS_GROUP=your-resource-group  # ❗ Your Azure resource group
+export AZURE_DEFAULTS_LOCATION=eastus2   # ❗ Your Azure location.
+```
+
+> 📖 Read [CLI configuration values and environment variables](https://learn.microsoft.com/en-us/cli/azure/azure-cli-configuration?source=recommendations#cli-configuration-values-and-environment-variables) for more ways to configure the `az` defaults.
+
+## 1. Create a Kubernetes cluster
+
+To get started, let's create a Kubernetes cluster in Microsoft Azure.
+You will need to pick a name for your cluster.
+Here, we will go with "test-cluster-1".
+Let us save it in an environment variable:
+
+```bash
+export CLUSTER=test-cluster-1
+```
+
+Now, create the cluster using the following command:
+
+```bash
+az aks create \
+    --name ${CLUSTER} \
+    --node-count 1 \
+    --node-vm-size "Standard_B2s" \
+    --load-balancer-sku basic \
+    --enable-oidc-issuer # ℹ️ This is required for workload identity federation and is only available when using the aks-preview extension.
```

Comment:
Thanks for writing the detailed docs! I'll try to have a closer look at the document soon.

There's a tiny thing that I've just spotted:
OIDC issuer has been promoted to GA in AKS [2022-10-16](https://github.com/Azure/AKS/blob/master/CHANGELOG.md#release-2022-10-16), so I think the `aks-preview` extension is only required in case Workload Identity itself is installed [as an extension](https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluster#register-the-enableworkloadidentitypreview-feature-flag).
```suggestion
    --enable-oidc-issuer # ℹ️ This is required for workload identity federation.
```

Reply:
Done.